CHANGES

```diff
@@ -0,0 +1,129 @@
+    - https://github.com/yaml/libyaml/pull/36 -- Fix -Wformat compilation errors in tests
+    - https://github.com/yaml/libyaml/pull/37 -- Update bug report URL in LibYAML
+    - https://github.com/yaml/libyaml/pull/38 -- Use AM_CPPFLAGS since autotools deprecated INCLUDE
+    - https://github.com/yaml/libyaml/pull/39 -- Update bug report URL in README
+    - https://github.com/yaml/libyaml/pull/41 -- Add travis and Makefile support for libyaml-test
+    - https://github.com/yaml/libyaml/pull/43 -- Add Dockerfile for Fedora 25
+    - https://github.com/yaml/libyaml/pull/44 -- WIP: Enable all warnings (-Wall) in libyaml and tests
+    - https://github.com/yaml/libyaml/pull/45 -- Fix typo
+    - https://github.com/yaml/libyaml/pull/47 -- Move travis script guts to separate file
+    - https://github.com/yaml/libyaml/pull/48 -- `yaml/libyaml-test` should become part of `yaml/libyaml`
+    - https://github.com/yaml/libyaml/pull/50 -- Add a GNUMakefile for immediate make targets
+    - https://github.com/yaml/libyaml/pull/53 -- Switch from test blacklist to whitelist
+    - https://github.com/yaml/libyaml/pull/55 -- Update defs for MingGW support on Windows
+    - https://github.com/yaml/libyaml/pull/58 -- Improve CMakeLists
+    - https://github.com/yaml/libyaml/pull/64 -- README: Update libyaml link
+    - https://github.com/yaml/libyaml/pull/69 -- Skip 5 tests in libyaml-emitter.list
+    - https://github.com/yaml/libyaml/pull/74 -- Forbid escaped singlequote in doublequotes
+    - https://github.com/yaml/libyaml/pull/76 -- Rewrite make test-suite
+    - https://github.com/yaml/libyaml/pull/77 -- Undefined PTRDIFF_MAX on HP-UX
+    - https://github.com/yaml/libyaml/pull/78 -- Fixed most compiler warnings -Wall -Wextra
+    - https://github.com/yaml/libyaml/pull/82 -- Move yaml-test-suite integration onto a separate branch.
+    - https://github.com/yaml/libyaml/pull/86 -- Fix problems in CI failures (travis and semaphore)
+    - https://github.com/yaml/libyaml/pull/87 -- appveyor.yml: add mingw-w64 builds
+    - https://github.com/yaml/libyaml/pull/88 -- add -no-undefined to src/Makefile.am
+    - https://github.com/yaml/libyaml/pull/89 -- Added alpine linux testing to dockerfiles
+    - https://github.com/yaml/libyaml/pull/93 -- remove need for PTRDIFF_MAX
+    - https://github.com/yaml/libyaml/pull/94 -- .gitignore: major cleanup
+    - https://github.com/yaml/libyaml/pull/120 -- Fix doc.
+
+0.1.7 2016-08-27
+
+    - Fixed segfault in yaml_string_write_handler.
+    - Fixed invalid simple key assertion.
+    - Fixed error handling in some examples (thank to Mathias Svensson).
+    - Removed obsolete VS project files.
+
+0.1.6 2014-03-26
+
+    - d1003a9 Fixed heap overflow in yaml_parser_scan_uri_escapes (Thanks
+      Ivan Fratric of the Google Security Team).
+    - 662f4be Added tag 0.1.5 for changeset a5142b24428b
+
+0.1.5 2014-02-03
+
+    - 303b455 Manually define PTRDIFF_MAX for VS C compiler.
```

Comment:
What do you think of changing shasums to urls like https://github.com/yaml/libyaml/commit/303b455?

Reply:
Hm, not sure. Can do it. In the future we shouldn't add commit lists anyway; I just added them because I didn't have a release summary.

Reply:
done